### ISSUE TITLE
Static properties can be defaulted

### ISSUE THREAD

**Issue by Ocramius** [Roave/BetterReflection#105]
https://github.com/Roave/BetterReflection/blob/83a9b0723cfda8ecd680aaad9911fa71ada532f3/src/Reflection/ReflectionProperty.php#L108-L110 seems incorrect, as a static property can have a default value as per following snippet:

```php
<?php

class Foo {
    public static $bar = 'tab';
}

$r = new ReflectionProperty('Foo', 'bar');

var_dump($r->isDefault());
```

See https://3v4l.org/LFk97

**Comment by asgrim** [comment]
Not sure how I reached that logic, I thought it was from reading [php-src](https://github.com/php/php-src/blob/master/ext/reflection/php_reflection.c#L5352) but evidently not... 

I'll also correct the phpdoc for this method ;)

**Comment by asgrim** [comment]
Closed via #106
